<?xml version="1.0" encoding="UTF-8" ?>
<launch>
    <arg name="pts_topics" default="[points]"/>
    <arg name="output_topic" default="grid_map/terrain"/>
    <arg name="robot_frame" default="base_link"/>
    <arg name="ground_frame" default="base_footprint"/>
    <arg name="max_age" default="0.2"/>

    <!-- Height map estimator node -->
    <node name="geom_hm_estimator" pkg="monoforce" type="geom_hm_estimator" output="screen">
        <rosparam subst_value="true">
            pts_topics: $(arg pts_topics)
            robot_frame: $(arg robot_frame)
            ground_frame: $(arg ground_frame)
            max_age: $(arg max_age)
        </rosparam>
        <remap from="grid_map/terrain" to="$(arg output_topic)"/>
    </node>
</launch>
